src/__tests__/main.test.ts

```diff
@@ -69,6 +69,25 @@ it('considers allow-one-liners.', () => {
   expect(mockSetFailed.mock.calls).toEqual([]);
 });
 
+it('considers skip-body-check.', () => {
+  (commitMessages.retrieve as any).mockImplementation(() => [
+    'Change SomeClass to OtherClass\n' +
+      '\n' +
+      'This replaces the SomeClass with OtherClass in all of the module ' +
+      'since Some class was deprecated.'
```

Comment:
```suggestion
      'Change SomeClass to OtherClass'
```

The repetition of the subject should trigger an error unless we set the `skip-body-check`.